src/mjolnir/pbfgraphparser.cc

```diff
@@ -2387,10 +2387,12 @@ struct graph_callback : public OSMPBF::Callback {
         if (tokens.size() == 2 && tmp.size()) {
 
           uint16_t mode = 0;
-          if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional") ||
-              boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
+          if (boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
```

Comment:
we have to respect `motorcar` as well though. I'm not sure what would be the best thing here, as that usually implies in some way that motorcycle & motorscooter are affected as well. If it's a `kTimedDenied` it usually doesn't count for motorcycles and they're still allowed, but if it's a `kTimedAllowed` it should also allow motorcycles. so maybe it `motorcar` needs to distinguish the type? That's what https://wiki.openstreetmap.org/wiki/Key:motorcar suggests.

Reply:
@gknisely I personally only looked at the wiki definition. IMO it's pretty clear that both motorcar and motorcycle are a subset of motor_vehicle, but motorcycle can't be a subset of motorcar, it's clearly a car.. If any mapper doesn't think so, I personally don't care, I can only stomach a certain level of OSM craziness :smile:

Reply:
fair enough